nearswap/src/deposit.rs

```diff
@@ -153,9 +171,28 @@ pub struct AccountDeposit {
 }
 
 impl AccountDeposit {
+    /// add given token to whitelist and set balance to 0.
+    /// Fails if not enough amount to cover new storage usage.
+    pub(crate) fn add_to_whitelist(&mut self, token_ids: &Vec<ValidAccountId>) {
+        for token_id in token_ids {
+            let t = token_id.as_ref();
+            if !self.tokens.contains_key(t) {
+                self.tokens.insert(t.clone(), 0);
+            }
+        }
+        self.assert_storage();
+    }
+
+    /// Remove `token_id` from this account whitelist and remove balance.
+    /// Panics if the `token_id` balance is not 0.
+    pub(crate) fn remove_from_whitelist(&mut self, token_id: &AccountId) {
+        let amount = self.tokens.remove(token_id).unwrap_or_default();
+        assert_eq!(amount, 0, "{}", ERR24_NON_ZERO_TOKEN_BALANCE);
+    }
     /**
     deposit `token`s. If this is a first depoisit, a new record is created and the minimum
     required storage is increased. */
+    /// Fails if account is not registered or if token isn't whitelisted.
```

Comment:
please merge the comment with the previous one. We shouldn't mix `///` with `/** ... */`

Reply:
ok